packages/remix-cloudflare-workers/package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/remix-run/remix/issues"
   },
   "dependencies": {
-    "@cloudflare/kv-asset-handler": "^0.1.3",
+    "@cloudflare/kv-asset-handler": "^0.2.0",
     "@remix-run/server-runtime": "1.0.6"
```

Comment:
`@remix-run/server-runtime` has `react` & `react-dom` as `peerDependency`.
Should we add them as dependencies or peerDependencies in this package too?